rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf

```diff
@@ -501,7 +501,7 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
 	setvar:tx.%{rule.id}-OWASP_CRS/WEB_ATTACK/XSS-%{matched_var_name}=%{tx.0}"
 
 
-SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "(?i:<[?]?import[ /+\t].*?implementation[ /+\t]*=)" \
+SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "<[?]?import[\s\/+\S]*?implementation[\s\/+]*?=" \
```

Comment:
1. Did you make regexp case sensitive here?
2. I'm not expert in how modsec processes regexp like `\/` - is it same as for `/` ? You are not escaping forward slash in any other regexes.
3. If you're using a construction `[\s...\S]`, it is ok to use `.` instead, as they are almost (by almost i mean newline, but that is not an issue for modsecurity) equivalent.